src/otx/v2/adapters/torch/lightning/anomalib/engine.py

```diff
@@ -205,16 +239,71 @@ def predict(
                 center_crop=center_crop,
                 normalization=normalization,
             )
-            dataset = InferenceDataset(img, image_size=image_size, transform=transform)
+            dataset = InferenceDataset(path=img, image_size=image_size, transform=transform)
             dataloader = DataLoader(dataset)
         elif isinstance(img, (DataLoader, LightningDataModule)):
             dataloader = [img]
         # Lightning Inferencer
-        return super().predict(
+        if model is None:
+            model = self.latest_model.get("model", None)
+        if checkpoint is None:
+            checkpoint = self.latest_model.get("checkpoint", None)
+
+        callbacks = callbacks if callbacks is not None else []
+        if not hasattr(self, "trainer"):
+            self._set_device(device=device)
+            if model is not None and hasattr(model, "callbacks"):
+                callbacks.extend(model.callbacks)
+            logger = self._update_logger(logger=logger, target_path=f"{self.timestamp}_predict")
+            self.trainer = Trainer(
+                logger=logger,
+                callbacks=callbacks,
+                **self.trainer_config,
+            )
+
+        if model is not None and checkpoint is not None:
+            self._load_checkpoint(model, checkpoint)
+        # Lightning Inferencer
+        return self.trainer.predict(
             model=model,
-            img=dataloader,
-            checkpoint=checkpoint,
-            device=device,
-            logger=logger,
-            callbacks=callbacks,
+            dataloaders=dataloader,
         )
+
+    def export(
+        self,
+        model: BaseOTXLightningModel | pl.LightningModule | None = None,
+        checkpoint: str | Path | None = None,
+        precision: _PRECISION_INPUT | None = None,
+        export_type: str = "OPENVINO",  # "ONNX" or "OPENVINO"
```

Comment:
`  # "ONNX" or "OPENVINO"` options are already defined in the docstring, so we don't need them here.
```suggestion
        export_type: str = "OPENVINO",
```

Reply:
I'll fix it.